rascal-vscode-extension/package.json

```diff
@@ -121,7 +121,22 @@
         "scopeName": "source.parametric-rascalmpl",
         "path": "./syntaxes/parametric.tmGrammar.json"
       }
-    ]
+    ],
+  "configuration": {
+      "title": "Rascal",
+      "properties": {
+        "rascal.interpreter.maxHeapSize": {
+          "type": ["number", "null"],
+          "default": null,
```

Comment:
why not have a default?

Reply:
There is already some computation happening in the plugin to try to pick a reasonable heap size based on the amount of available RAM. If no heap size is specified, we just fall back to that existing computation.